From 612388c42d00ab8eba1d9626a7da33a18c724d76 Mon Sep 17 00:00:00 2001
From: bartoszek <bartus@yopmail.com>
Date: Sun, 15 Jan 2017 22:19:18 +0100
Subject: [PATCH] add missing linker flag for linux

also extend post link file copying to include linux build ( linking is somewhat strange but wont work without )
---
 src/meshlabplugins/filter_screened_poisson/filter_screened_poisson.pro | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/src/meshlabplugins/filter_screened_poisson/filter_screened_poisson.pro b/src/meshlabplugins/filter_screened_poisson/filter_screened_poisson.pro
index 814bd31..a93f67a 100644
--- a/src/meshlabplugins/filter_screened_poisson/filter_screened_poisson.pro
+++ b/src/meshlabplugins/filter_screened_poisson/filter_screened_poisson.pro
@@ -2,6 +2,7 @@ include (../../shared.pri)
 
 macx:QMAKE_CXX = clang++-mp-3.9
 macx:QMAKE_LFLAGS += -L/opt/local/lib/libomp -lomp
+linux:QMAKE_LFLAGS += -lgomp
 macx:QMAKE_CXXFLAGS_RELEASE+= -O3 -DRELEASE -funroll-loops -ffast-math  -Wno-sign-compare -Wno-unused-parameter
 
 QMAKE_CXXFLAGS+=-fopenmp
@@ -22,4 +23,4 @@ DEFINES += FOR_RELEASE
 
 #PRE_TARGETDEPS += ./filter_screened_poisson.xml
 macx:QMAKE_POST_LINK = "cp "$$_PRO_FILE_PWD_/$$TARGET".xml ../../distrib/plugins/"$$TARGET".xml"
-
+linux:QMAKE_POST_LINK = "cp "$$_PRO_FILE_PWD_/$$TARGET".xml ../../distrib/plugins/"$$TARGET".xml; cd ../../distrib/plugins/ ; ln -s "$$TARGET".xml lib"$$TARGET".xml"
